fetcher.drush.inc

```diff
@@ -449,6 +449,8 @@ function drush_fetcher_delete($site_name) {
   }
   try {
     $site = fetcher_get_site($site_name);
+    $default_dir = $site['site.webroot'] . '/sites/default';
+    chmod($default_dir, 0777);
```

Comment:
In theory, fetcher is designed to be system agnostic. I think this should be performed via a call to (what would probably be a new method) in the [system class](https://github.com/tizzo/fetcher/blob/7.x-1.x/lib/Fetcher/System/Posix.php).

Want to take a stab at doing that? Then you could do `$site['system']->setPermission($site['site.directory'], 0777);` instead. Note also that fetcher can work with sites other than default, so you should be looking that up as done in the above.